integration_tests/integration/serviceProviderReferrals.spec.js

```diff
@@ -1943,60 +1943,6 @@ describe('Service provider referrals dashboard', () => {
       })
 
       describe('with appointments in the past', () => {
-        describe('that happened earlier today', () => {
-          it('presents the feedback form as soon as the appointment is confirmed', () => {
```

Comment:
can we change this test to validate the current expected behaviour instead of removing it?